.stickler.yml

```diff
@@ -0,0 +1,2 @@
+linters:
+  stylelint:
```

Comment:
Please modify this file to be the same as the one provided by Microverse, you can use this [link](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml).